presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationUtils.java

```diff
@@ -126,14 +123,10 @@ public static void mergeRegressionState(RegressionState state, RegressionState o
         state.setSumXSquare(state.getSumXSquare() + otherState.getSumXSquare());
     }
 
-    public static Type getOutputType(@Nullable Method outputFunction, AccumulatorStateSerializer<?> serializer, TypeManager typeManager)
+    public static TypeSignature getOutputTypeSignature(@Nullable Method outputFunction)
```

Comment:
Since this is only used in one place and is so simple now, I would just inline it and remove the method